### ISSUE TITLE
Epoch vs Steps_per_epoch vs Batch_size

### ISSUE THREAD

**Issue by Densus100** [Kamlesh364/Mask-RCNN-TF2.7.0-keras2.7.0#9]
Hello guys, I'm trying to re-train the Mask RCNN to fit into my dataset, but i stuck at the config and really confused about some parameters. **Please anyone that maybe able to help me distinguish the difference between :**

1. EPOCH
2. STEPS_PER_EPOCH
3. BATCH_SIZE

Explanation with example like the numbers of the dataset and how many iteration that the models do between those parameters will be much appreciated. Thank you :)

**Comment by Kamlesh364** [comment]
@Densus100 
Please go through [this](https://stackoverflow.com/questions/38340311/what-is-the-difference-between-steps-and-epochs-in-tensorflow).

hope you'll find this insightful.

**Comment by Densus100** [comment]
Thank you. I'll Close this issue then :)
